python/huggingfaceserver/pyproject.toml

```diff
@@ -12,9 +12,9 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 kserve = { path = "../kserve", extras = ["storage"], develop = true }
-transformers = "~4.36.0"
+transformers = "~4.37.0"
 torch = "~2.1.2"
-vllm = { version = "^0.2.6", optional = true }
+vllm = "~0.2.7"
```

Comment:
We can still keep it optional as install it as default in the dockerfile as the downside of doing this is that I can't develop on my Mac

Reply:
In this case, will the baked image have vllm installed or some manual intervention is needed?

I had also removed code related to ImportError as this was mandatory. Would need to rollback those changes right?

Reply:
in the dockerfile we can install vllm,  but would like leave as an option to disable vllm for local development. Also you won’t pass all the tests as CI does not have gpu.